src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsCard.component.ts

```diff
@@ -97,6 +107,10 @@ export class NotebookViewsCardComponent implements OnInit {
 		this.onChange.emit({ cell: this.cell, event: event });
 	}
 
+	get modal(): boolean {
```

Comment:
This doesn't seem clear to me - what is "modal" supposed to mean in this case? Why not just expose awaitingInput as a public readonly property?

Reply:
I renamed it to displayInputModal to make it a bit more explicit. I wanted to not tie the display of the modal directly to awaitingInput in case the criteria for displaying the modal changed later on.